client/app/account/billing/payments/request/billing-payments-request.html

```diff
@@ -0,0 +1,48 @@
+<div class="billing-payments-request">
+
+    <oui-back-button data-title="{{ :: ('billing_payments_request_title' | translate) }}"
+                     data-href="{{ $ctrl.$state.href('^') }}">
+    </oui-back-button>
+
+    <div class="text-center"
+         data-ng-if="$ctrl.loading.init">
+        <oui-spinner></oui-spinner>
+     </div>
+
+    <div data-ng-if="!$ctrl.loading.init">
+        <div data-ovh-alert="billing_payment_request"></div>
+
+        <div data-ng-if="!$ctrl.requestPending">
+            <div class="mb-4">
```

Comment:
You can apply your `data-ng-if="!$ctrl.requestPending"` here if you want:)

Reply:
Nope, cause datagrid is hidden during loading